client/dvm/rpc/src/eth.rs

```diff
@@ -900,12 +1046,21 @@ where
 		};
 		let data = data.map(|d| d.0).unwrap_or_default();
 
+		let api_version = if let Ok(Some(api_version)) =
+			api.api_version::<dyn EthereumRuntimeRPCApi<B>>(&BlockId::Hash(hash))
+		{
+			api_version
+		} else {
+			return Err(internal_err(format!(
+				"failed to retrieve Runtime Api version"
+			)));
+		};
+
 		match to {
 			Some(to) => {
-				let info = self
-					.client
-					.runtime_api()
-					.call(
+				if api_version == 1 {
+					#[allow(deprecated)]
+					let info = api.call_before_version_2(
```

Comment:
https://github.com/darwinia-network/frontier/blob/darwinia-v0.12.1/primitives/rpc/src/lib.rs#L52-L79. We need to upgrade the api version since the core block type updated in this pr.